modules/api/src/main/java/org/apache/fluo/api/service/FluoService.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.fluo.api.service;
+
+/**
+ * Shared interface for Fluo services
+ */
+public interface FluoService {
+
+  /**
+   * Starts service
```

Comment:
Should document expected behavior if service fails to start.  Should also document if this expected to be synchronous or async.